src/components/gds-menu-item/gds-menu-item.scss

```diff
@@ -19,52 +14,36 @@
   @include desktopOnly {
     padding: 0 var(--spacing-s);
   }
-}
-
-// Vertical direction explicitly defined.
-:host-context(.vertical) {
-  & .content {
-    padding: 0 var(--spacing-m);
-  }
-}
-
-// Horizontal direction explicitly defined.
-:host-context(.horizontal) {
-  & .content {
-    padding: 0 var(--spacing-s);
-  }
-}
 
-.item {
   // For historical reasons, fallback to gds-label size="l"
   font-family: var(--gds-menu-item-font-family, var(--label-font-family));
   font-size: var(--gds-menu-item-font-size, var(--label-l-font-size));
   line-height: var(--gds-menu-item-line-height, var(--label-l-line-height));
   letter-spacing: var(--gds-menu-item-letter-spacing, var(--label-l-letter-spacing));
   font-weight: var(--gds-menu-item-font-weight, var(--label-l-font-weight));
   text-transform: var(--gds-menu-item-text-transform, var(--label-text-transform));
+  // Provides a possibility to offset text content when the font has incorrect
+  // vertical alignment for glyphs.
+  transform: var(--gds-menu-item-transform);
```

Comment:
This is needed to align the font with the dropdown icon in submenus using the font GDS has. If it was arial font with a correct glyph alignment this wouldn't be needed.

To be honest it sort of sucks that our design system is so opinionated that it sets this transform to 0.15em because of the font it uses...

I actually vote GDS should use Helvetica or some other readily available font.

The problem are these 2px below the text misaligning everything when centered.

<img width="246" alt="Screen Shot 2021-03-28 at 15 01 58" src="https://user-images.githubusercontent.com/302736/112762554-94f83580-8fd6-11eb-8a7e-18cff7d38cf7.png">

Reply:
Should we just slot the submenu icon rather than try to offset the item text? Also then we could use what ever icon.

Reply:
We could do that too but I think Gilroy font (and other fonts) anyways might need this transform since they're the ones that aren't centered–the submenu icon is.

Too bad there's any `font-baseline-offset: 0.15em` or similar way to fix it for fonts.